### PR TITLE
feat(tutorial): complete step 5

### PR DESCRIPTION
**Pull request by obro**
See deployed on Bluemix: http://carbon-tutorial-obro.mybluemix.net/#/

### CODE REVIEW

**Comment by obro** [comment]
@kellychurchill Apologies! Added here and in the description.

http://carbon-tutorial-obro.mybluemix.net/#/

File: src/index.scss [hunk 1]
```diff
@@ -1,9 +1,19 @@
-$feature-flags: (
```
Comment (kellychurchill) [inline src/index.scss]: The feature-flags mapping should not be deleted.

**Comment by obro** [comment]
@kellychurchill Oops, not sure how I missed that! PR and deploy both updated.

**Comment by kellychurchill** [comment]
👍  Looks good! 
Feel free to stop this running application on IBM Cloud to avoid the possibility of incurring any charges 💵

**Pull request closed without merging**